test/nullablevector.jl

```diff
@@ -152,9 +155,21 @@ module TestNullableVector
                   NullableArray([2:12...]))
     @test isequal(append!(X, [Nullable(13), Nullable(14)]),
                   NullableArray([2:14...]))
-    @test isequal(append!(X, [Nullable(15), Nullable()]),
+    @test isequal(append!(X, [Nullable(15), Nullable{Int64}()]),
                   NullableArray([2:16...], vcat(fill(false, 14), true)))
 
+    #--- test Base.prepend!
+
+    # Base.prepend!(X::NullableVector, items::AbstractVector)
+
+    XX = NullableArray([3:12...])
```

Comment:
Just call it `X` again, this variable is used several times anyway in this file.